modules/gui/include/visp3/gui/vpColorBlindFriendlyPalette.h

```diff
@@ -0,0 +1,203 @@
+/****************************************************************************
+ *
+ * ViSP, open source Visual Servoing Platform software.
+ * Copyright (C) 2005 - 2023 by Inria. All rights reserved.
+ *
+ * This software is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ * See the file LICENSE.txt at the root directory of this source
+ * distribution for additional information about the GNU GPL.
+ *
+ * For using ViSP with software that can not be combined with the GNU
+ * GPL, please contact Inria about acquiring a ViSP Professional
+ * Edition License.
+ *
+ * See http://visp.inria.fr for more information.
+ *
+ * This software was developed at:
+ * Inria Rennes - Bretagne Atlantique
+ * Campus Universitaire de Beaulieu
+ * 35042 Rennes Cedex
+ * France
+ *
+ * If you have questions regarding the use of this file, please contact
+ * Inria at visp@inria.fr
+ *
+ * This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
+ * WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
+ * 
+ * 
+ *
+ * Description:
+ * Palette of colors that is said to be color-blind people friendly.
+ * This palette has been found on https://jfly.uni-koeln.de/color/#see
+ *
+ * Authors:
+ * Romain LAGNEAU
+ *
+ *****************************************************************************/
+
+#ifndef VPCOLORBLINDFRIENDLYPALETTE_H
+#define VPCOLORBLINDFRIENDLYPALETTE_H
+
+#include<visp3/core/vpColor.h>
+
+/**
+ * \brief Class that furnishes a set of colors that color blind people
+ * should be able to distinguish one from another.
+ */
+class VISP_EXPORT vpColorBlindFriendlyPalette
+{
+public:
+  /**
+   * \brief Enum that list the different available colors.
+   */
+  typedef enum class Palette{
+    Black = 0,     /*!< Color that looks like black for non-colorblind people.*/
+    Orange = 1,    /*!< Color that looks like orange for non-colorblind people.*/
+    SkyBlue = 2,   /*!< Color that looks like sky blue for non-colorblind people.*/
+    Green = 3,     /*!< Color that looks like green for non-colorblind people.*/
+    Yellow = 4,    /*!< Color that looks like yellow for non-colorblind people.*/
+    Blue = 5,      /*!< Color that looks like blue for non-colorblind people.*/
+    Vermillon = 6, /*!< Color that looks like vermillon for non-colorblind people.*/
+    Purple = 7,    /*!< Color that looks like purple for non-colorblind people.*/
+    COUNT = 8      /*!< Number of colors the palette proposes.*/
+  }Palette;
+
+  static std::vector<std::string> g_paletteNames; /*!< Vector that lists the names of the different available colors.*/
```

Comment:
Class member name should be homogeneous. Instead of using `g_` as prefix, use `s_` for static members and `m_` for other classical vars